src/algebra/non_unital_alg_hom.lean

```diff
@@ -57,11 +57,9 @@ structure non_unital_alg_hom [monoid R]
 attribute [nolint doc_blame] non_unital_alg_hom.to_distrib_mul_action_hom
 attribute [nolint doc_blame] non_unital_alg_hom.to_mul_hom
 
-initialize_simps_projections non_unital_alg_hom (to_fun → apply)
-
 namespace non_unital_alg_hom
 
-variables {R A B C} [semiring R]
+variables {R A B C} [monoid R]
```

Comment:
I think this will have made `simps` happy

Reply:
Boom! Nice find.